AcademyResidentInformationApi.Tests/V1/Gateways/AcademyGatewayTests.cs

```diff
@@ -233,10 +238,7 @@ public void GetAllResidentsWithNameAndPostCodeQueryParameterReturnsMatchingResid
         public void GetAllResidentsWithPostCodeQueryParameterIgnoresFormatting(string postcode)
         {
             var databaseEntity = AddPersonRecordToDatabase();
-
-            var address = TestHelper.CreateDatabaseAddressForPersonId(databaseEntity.ClaimId, databaseEntity.HouseId, postcode);
-            AcademyContext.Addresses.Add(address);
-            AcademyContext.SaveChanges();
+            var address = AddAddressToDatabase(databaseEntity.ClaimId, databaseEntity.HouseId, postcode: postcode);
```

Comment:
The `postcode: postcode` argument here is an argument with a label? Is [this](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#named-arguments) the right bit of the docs for me to read up?

Reply:
Yep and yes to the docs as well :)